core/api.go

```diff
@@ -64,6 +66,14 @@ func (a *api) Start() {
 		g.Writer.WriteHeader(http.StatusNoContent)
 	})
 
+	// Allows all origins
+	// TODO: Do not use this in production, needs to be configurable
+	config := cors.DefaultConfig()
```

Comment:
This should probably really be part of a different PR... required to use any of our api commands in CORS context.

Reply:
cool, can you open an issue to make this part of the textile config file?

Reply:
added